src/data/finset/card.lean

```diff
@@ -42,6 +42,7 @@ variables {s t : finset α} {a b : α}
 def card (s : finset α) : ℕ := s.1.card
 
 lemma card_def (s : finset α) : s.card = s.1.card := rfl
+@[simp] lemma card_val (s : finset α) : s.1.card = s.card := rfl
```

Comment:
I'm not really sold on any of these new simp lemmas; and they certainly seem orthogonal to the titular results of the PR

Reply:
Consider `prod_le_pow_card` in `algebra.big_operators.order`, which can now be proved with `simp` instead of `simpa` because of the new simp lemma here.